### PR TITLE
rough draft of visualization 'About'

### PR DESCRIPTION
**Pull request by JamesKunstle**
Fill in a new edit of visualization descriptions.

### CODE REVIEW

File: pages/overview/visualizations/active_drifting_contributors.py [hunk 1]
```diff
@@ -30,10 +30,11 @@
                     [
                         dbc.PopoverHeader("Graph Info:"),
                         dbc.PopoverBody(
-                            "Select the intervals that make sense for your community!\n\
-                            <ACTIVE> contributors that have a contribution within the drifting parameter.\n\
```
Comment (cdolfi) [inline pages/overview/visualizations/active_drifting_contributors.py:34]: Feel like we need some version on this either in the description or info page

**Review by cdolfi — CHANGES_REQUESTED** [review]
very minor comments but overall looks awesome

File: pages/company/visualizations/unqiue_domains.py [hunk 2]
```diff
@@ -30,7 +30,14 @@
                 dbc.Popover(
                     [
                         dbc.PopoverHeader("Graph Info:"),
-                        dbc.PopoverBody("This graph counts the number of UNIQUE emails by domains"),
+                        dbc.PopoverBody(
+                            """
+                            Visualizes the population of unique commit email addresses per represented domain.\n
+                            e.g. if there are 100 commit contributors and 50 have a '@gmail.com' email address,\n
```
[the remark this answers is not in the document]
Comment (JamesKunstle) [reply pages/company/visualizations/unqiue_domains.py:36]: ack

File: pages/overview/visualizations/active_drifting_contributors.py (hunk 1 above)
Comment (JamesKunstle) [reply pages/overview/visualizations/active_drifting_contributors.py:34]: The definition is on the Info page in the other PR.

File: pages/company/visualizations/unqiue_domains.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (JamesKunstle) [reply pages/company/visualizations/unqiue_domains.py]: lmk what you think about the updated phrasing

File: pages/company/visualizations/unqiue_domains.py [hunk 3]
```diff
@@ -30,7 +30,14 @@
                 dbc.Popover(
                     [
                         dbc.PopoverHeader("Graph Info:"),
-                        dbc.PopoverBody("This graph counts the number of UNIQUE emails by domains"),
+                        dbc.PopoverBody(
+                            """
+                            Visualizes the population of unique commit email addresses per represented domain.\n
+                            e.g. if there are 100 distrinct commit contributors and 50 use an '@gmail.com' email address,\n
```
Comment (cdolfi) [inline pages/company/visualizations/unqiue_domains.py:36]: distinct

**Review by cdolfi — CHANGES_REQUESTED** [review]
Small spelling change and its good to go

File: pages/company/visualizations/unqiue_domains.py (hunk 3 above)
Comment (JamesKunstle) [reply pages/company/visualizations/unqiue_domains.py]: done

**Comment by JamesKunstle** [comment]
@cdolfi merge whenever you can, no rush.

**Review by cdolfi — APPROVED** [review]
lgtm

**Pull request merged**